libs/wire-api/src/Wire/API/User/RichInfo.hs

```diff
@@ -63,6 +63,12 @@ import Wire.API.Arbitrary (Arbitrary (arbitrary))
 --------------------------------------------------------------------------------
 -- RichInfo
 
+-- | Represents all the ways we can recieve 'RichInfo' from a SCIM peer.
+-- 'richInfoMap' represents fields given under 'richInfoMapURN'. It is expected
+-- to be a JSON object. 'richInfoAssocList' represents fields given under
+-- 'richInfoAssocListURN'. It is expected to be an assoc list. Use the much
```

Comment:
This comment is a bit confusing and can be read as " use `richInfoAssocListURN` if you can" which is the opposite of what we want as it isn't SCIM-compliant

Reply:
https://github.com/zinfra/backend-issues/issues/1627